src/sql/command.py

```diff
@@ -17,8 +18,12 @@ def __init__(self, magic, user_ns, line, cell) -> None:
 
         self.args = parse.magic_args(magic.execute, line)
 
+        # self.args.line (everything that appears after %sql/%%sql in the first line)
+        # is splited in tokens (delimited by spaces), this checks if we have one arg
+        one_arg = len(self.args.line) == 1
+
         if (
-            len(self.args.line) == 1
+            one_arg
             and self.args.line[0] in user_ns
```

Comment:
you can do the same here as the one_arg condition

Reply:
I cannot.  `self.args.line[0]` will break if `selg.args.line` is an empty list. passing it after the `len(self.args.line)` ensures that line is only evaluated when there's at least one element

Reply:
Good point.